### DISCUSSION TITLE
plot_pair: coords for pairs of variables.

### DISCUSSION THREAD

**Discussion by pscicluna** [pydata/xarray#5063] [Q&A]
HI! I'm trying to understand how I can make the `coords` keyword of many arviz plotting routines to behave in a certain way. I hope explain the problem well enough, I'll gladly try to clarify if not.

To outline my problem, I'm inferring the contents of a correlation matrix with PyMC3. Now, because the matrix is symmetrical, I only want to use the upper (or lower, if it makes a difference) off-diagonal elements, that is to use elements `[0,1]`, `[1,2]`, `[0,2]` if it's a 3-dimensional problem. 

However, plot_pair seems to interpret `coords` as a "slice" along a dimension, such that if I set `coords` to [0,1] on dimension one of the correlation matrix and [1,2] on dimension two, I will also get the plots for [1,1]. Is there any way that I can leave this one out automatically, or do I have to manually iterate over a set of axes to get this?

**Comment by keewis** [comment]
@pscicluna, this looks like a question about `arviz`, not `xarray` (`xarray` doesn't have a `plot_pair` function / method). You might get a quicker response if you ask this question on their [repository](https://github.com/arviz-devs/arviz/discussions).

**Reply by pscicluna** [reply]
Sorry, you're right - I was following links from one issue to another and ended up on the wrong repo. It's _tangentially_ related since arviz is using an xarray internally, but hopefully I'll get a quicker answer from the arviz devs. My apologies!

**Reply by keewis** [reply]
no worries!
